ext/dom/tests/domobject_debug_handler.phpt

```diff
@@ -28,8 +28,8 @@ DOMDocument Object
     [actualEncoding] => 
     [encoding] => 
     [xmlEncoding] => 
-    [standalone] => 1
-    [xmlStandalone] => 1
+    [standalone] => 
+    [xmlStandalone] => 
```

Comment:
So, I'm going to be annoying, but could you maybe change the ``print_r()`` to a ``var_dump()`` so there are not the trailing white spaces. As this will cause issues with IDEs...